### PR TITLE
BL-1182 Bug with purchase order records

### PR DESCRIPTION
**Pull request by sensei100**
- A recent change resulted in purchase order titles being suppressed.  This updates the logic so that those records are included in the indexing process.

### CODE REVIEW

File: spec/fixtures/marc_files/lost_missing_technical.xml [hunk 1]
```diff
@@ -474,4 +474,15 @@
...
 	</record>
+	<record>
+		<!-- 11. Purchase order fields are present -->
+		<controlfield tag="001">11</controlfield>
+		<datafield ind1=" " ind2=" " tag="902">
+			<subfield code="a">test</subfield>
+		</datafield>
+	</record>
+	<record>
+		<!-- 12. Purchase order fields are present -->
```
Comment (bibliotechy) [inline spec/fixtures/marc_files/lost_missing_technical.xml:485]: Should this be
```suggestion
		<!-- 12. Purchase order fields are not present -->
```

**Comment by sensei100** [comment]
@bibliotechy Thanks for catching that.

**Comment by ebtoner** [comment]
@sensei100 @bibliotechy I think a bit of additional logic may need to be added beyond just looking for 902a. I think we also need to specifically look for the value "EBC-POD" ... @htomren, is that correct?

**Comment by ebtoner** [comment]
See https://github.com/tulibraries/cob_index/blob/fe148b8e2eaa365788bf8532858d5f253246a101/lib/cob_index/macros/custom.rb#L689-L693

**Comment by ebtoner** [comment]
Confirmed by Holly (over Slack) that it should include the EBC-POD logic.

**Comment by sensei100** [comment]
@bibliotechy The logic has been updated, so this could be merged now.

**Pull request merged**